libraries/lib-telemetry/TelemetryManager.cpp

```diff
@@ -277,7 +277,8 @@ void Terminate ()
 
     // TrackingService may block the thread for a short period
     // of time if required.
-    telemetryManager.UserTtrackingService->reportFinished();
+    if (telemetryManager.UserTtrackingService != nullptr)
+        telemetryManager.UserTtrackingService->reportFinished();
```

Comment:
Can I start another review?  Yes I can.  This is just a test of GitHub.

And it seems you found a bug in a 3pl to fix with a null check.  Did it crash for you in testing?